xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2105,6 +2105,7 @@ xe.activity.noentries=There are no activities in the stream
 xe.activity.action.create=created the page
 xe.activity.action.delete=deleted the page
 xe.activity.action.update=edited the page
+xe.activity.action.BlogPostPublishedEvent=published the post
```

Comment:
Could you change that to "published a blog post"?

Thanks

Reply:
done